src/httpserver.cpp

```diff
@@ -388,6 +388,12 @@ static bool HTTPBindAddresses(struct evhttp* http)
             if (i->first.empty() || (addr.has_value() && addr->IsBindAny())) {
                 LogPrintf("WARNING: the RPC server is not safe to expose to untrusted networks such as the public internet\n");
             }
+            // Set the no-delay option (disable Nagle's algorithm) on the TCP socket.
+            evutil_socket_t fd = evhttp_bound_socket_get_fd(bind_handle);
+            int nOne = 1;
+            if (setsockopt(fd, IPPROTO_TCP, TCP_NODELAY, (sockopt_arg_type)&nOne, sizeof(int)) == SOCKET_ERROR) {
+                LogPrintf("WARNING: Unable to set TCP_NODELAY on RPC server socket, continuing anyway\n");
```

Comment:
`LogPrintf` is deprecated. See https://github.com/bitcoin/bitcoin/pull/29641

Reply:
Many thanks! Fixed in f14484184f8cae134f65801647280c2c1f4930b2.